api/services/ConversionService.js

```diff
@@ -58,7 +58,15 @@ module.exports = {
             mathJaxNodeOptions = extend(options, {timeout: 100 * 1000});
 
         mathjaxNode.typeset(options, function (data) {
-            done(data);
+            if (options.png) {
+                var mathjaxNode2 = require('mathjax-node-svg2png');
```

Comment:
~~Suggestion: Move the import up to the top of the file... oh, the current code does not do that so nevermind 😄~~